src/logic/operate.js

```diff
@@ -0,0 +1,25 @@
+import Big from 'big.js';
+
+const operate = (number1, number2, operation) => {
+  if (!number1 || !number2 || !operation) return '0';
+
+  const numberOne = Number(number1);
+  const numberTwo = Number(number2);
+
+  switch (operation) {
+    case '+':
+      return (Big(numberOne) + Big(numberTwo)).toString();
+    case '-':
+      return (Big(numberOne) - Big(numberTwo)).toString();
+    case '/':
+      return (Big(numberOne) / Big(numberTwo)).toString();
+    case '*':
+      return (Big(numberOne) * Big(numberTwo)).toString();
+    case '%':
+      return ((Big(numberOne) * Big(numberTwo)) / 100).toString();
+    default:
```

Comment:
- [ ] [REQUIRED] Good job by wrapping numberOne and numberTwo in Big class but it will be also perfect if you use functions provided by `Big` package like `plus`, `minus`, `times`, `div` and `modifier` 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png))